README.md

```diff
@@ -82,3 +84,36 @@ See our FAQ for more details:
 
 Once the node template is running locally, you can connect to it with frontends like [Contracts UI](https://contracts-ui.substrate.io/#/?rpc=ws://127.0.0.1:9944) or [Polkadot-JS Apps](https://polkadot.js.org/apps/#/explorer?rpc=ws://localhost:9944) and interact with your chain.
 
+## How to synchronize with Substrate
+
+- [ ] Check Substrate's [`node-template`](https://github.com/paritytech/substrate/commits/master/bin/node-template)
+      for new commits since the last time someone synchronized this
+			repository with Substrate. The commit hash of the last sync is
+			mentioned in this readme.
+- [ ] Apply each commit that happened in this `node-template` folder
+      since the last sync.
+- [ ] Check commits for [`pallet-contracts`](https://github.com/paritytech/substrate/tree/master/frame/contracts)
+      since the last time someone synchronized this repository with Substrate.
+      You can limit yourself to check commits that modified the
+			`pallet-contracts` configuration in Substrate's folder `bin/node/`.
+			This folder includes the `pallet-contracts`, so whenever a breaking
+			change is introduced in the pallet it will be reflected in `bin/node/`.
```

Comment:
To what end would you check the commits?

Reply:
👆

Reply:
Yeah, so it's tricky. I mean it depends on each commit. Usually you can say "just stuff that affects `bin/node/`. But there could also be things that are only relevant to this "developer" node.

Reply:
I left it open to interpretation, do you have a better idea how to formulate it?